default_settings.json

```diff
@@ -0,0 +1,5254 @@
+
+	// Controls whether the editor should automatically adjust the indentation when users type, paste, move or indent lines.
+	//  - none: The editor will not insert indentation automatically.
+	//  - keep: The editor will keep the current line's indentation.
+	//  - brackets: The editor will keep the current line's indentation and honor language defined brackets.
+	//  - advanced: The editor will keep the current line's indentation, honor language defined brackets and invoke special onEnterRules defined by languages.
+	//  - full: The editor will keep the current line's indentation, honor language defined brackets, invoke special onEnterRules defined by languages, and honor indentationRules defined by languages.
+	"editor.autoIndent": "full",
+
+	// Controls whether the editor should automatically surround selections when typing quotes or brackets.
+	//  - languageDefined: Use language configurations to determine when to automatically surround selections.
+	//  - quotes: Surround with quotes but not brackets.
+	//  - brackets: Surround with brackets but not quotes.
+	//  - never
+	"editor.autoSurround": "languageDefined",
+
+	// Controls whether bracket pair colorization is enabled or not. Use `workbench.colorCustomizations` to override the bracket highlight colors.
+	"editor.bracketPairColorization.enabled": false,
+
+	// Controls whether each bracket type has its own independent color pool.
+	"editor.bracketPairColorization.independentColorPoolPerBracketType": false,
+
+	// Code action kinds to be run on save.
+	"editor.codeActionsOnSave": {},
+
+	// Controls whether the editor shows CodeLens.
+	"editor.codeLens": true,
+
+	// Controls the font family for CodeLens.
+	"editor.codeLensFontFamily": "",
+
+	// Controls the font size in pixels for CodeLens. When set to `0`, 90% of `editor.fontSize` is used.
+	"editor.codeLensFontSize": 0,
+
+	// Controls whether the editor should render the inline color decorators and color picker.
+	"editor.colorDecorators": true,
+
+	// Enable that the selection with the mouse and keys is doing column selection.
+	"editor.columnSelection": false,
+
+	// Controls if empty lines should be ignored with toggle, add or remove actions for line comments.
+	"editor.comments.ignoreEmptyLines": true,
+
+	// Controls whether a space character is inserted when commenting.
+	"editor.comments.insertSpace": true,
+
+	// Controls whether syntax highlighting should be copied into the clipboard.
+	"editor.copyWithSyntaxHighlighting": true,
+
+	// Control the cursor animation style.
+	"editor.cursorBlinking": "blink",
+
+	// Controls whether the smooth caret animation should be enabled.
+	"editor.cursorSmoothCaretAnimation": false,
+
+	// Controls the cursor style.
+	"editor.cursorStyle": "line",
+
+	// Controls the minimal number of visible leading and trailing lines surrounding the cursor. Known as 'scrollOff' or 'scrollOffset' in some other editors.
+	"editor.cursorSurroundingLines": 0,
```

Comment:
I think this value should be something like `10` — especially useful for teaching, so code doesn't get too far down at the bottom of the screen (otherwise have to add newlines manually).